kostal/README.md

```diff
@@ -63,35 +62,97 @@ daily power. Additionally it shows the volts and watts for the phases.
 # items/my.conf
 [solar]
     [[status]]
+        name = Wechselrichter Status
         type = str
-        kostal = status
-    [[current]]
+        kostal = operation_status
+    [[dcpower]]
+        name = Gleichspannungsleitung gesamt
+        type = num
+        kostal = dctot_w
+    [[dc1_v]]
+        name = DC-Strang 1 Spannung
+        type = num
+        kostal = dc1_v
+    [[dc1_a]]
+        name = DC-Strang 1 Strom
+        type = num
+        kostal = dc1_a
+    [[dc1_w]]
+        name = DC-Strang 1 Leistung
+        type = num
+        kostal = dc1_w
+    [[dc2_v]]
+        name = DC-Strang 2 Spannung
         type = num
-        kostal = power_current
-    [[total]]
+        kostal = dc2_v
+    [[dc2_a]]
+        name = DC-Strang 2 Strom
         type = num
-        kostal = power_total
-    [[day]]
+        kostal = dc2_a
+    [[dc2_w]]
+        name = DC-Strang 2 Leistung
         type = num
-        kostal = power_day
-    [[l1v]]
+        kostal = dc2_w
+    [[actot_w]]
+        name = Wechselspannungsleistung gesamt
         type = num
-        kostal = l1_watt
-    [[l1w]]
+        kostal = actot_w
+    [[actot_cos]]
+        name = Cos phi
         type = num
-        kostal = l1_watt
-    [[l2v]]
+        kostal = actot_cos
+    [[actot_limitation]]
+        name = Abregelungsfaktor
         type = num
-        kostal = l2_watt
-    [[l2w]]
+        kostal = actot_limitation
+    [[ac1_v]]
+        name = Phase 1 Spannung
         type = num
-        kostal = l2_watt
-    [[l3v]]
+        kostal = ac1_v
+    [[ac1_a]]
+        name = Phase 1 Strom
         type = num
-        kostal = l3_watt
-    [[l3w]]
+        kostal = ac1_a
+    [[ac1_w]]
+        name = Phase 1 Leistung
         type = num
-        kostal = l3_watt
+        kostal = ac1_w
+    [[ac2_v]]
+        name = Phase 2 Spannung
+        type = num
+        kostal = ac2_v
+    [[ac2_a]]
+        name = Phase 2 Strom
+        type = num
+        kostal = ac2_a
+    [[ac2_w]]
+        name = Phase 2 Leistung
+        type = num
+        kostal = ac2_w
+    [[ac3_v]]
+        name = Phase 3 Spannung
+        type = num
+        kostal = ac3_v
+    [[ac3_a]]
+        name = Phase 3 Strom
+        type = num
+        kostal = ac3_a
+    [[ac3_w]]
+        name = Phase 3 Leistung
+        type = num
+        kostal = ac3_w
+    [[yield_day_wh]]
+        name = Gesamtleistung heute
+        type = num
+        kostal = yield_day_wh
+    [[yield_tot_kwh]]
+        name = Gesamtleistung total
+        type = num
+        kostal = yield_tot_kwh
+    [[operationtime_h]]
+        name = Betriebsstunden
+        type = num
+        kostal = operationtime_h
 </pre>
```

Comment:
If we do not want to use the same keys/names for `kostal` setting we should provide an example for both types (html and json).